app/views/webhook_deliveries/show.html.haml

```diff
@@ -35,6 +35,13 @@
       %th Configured Webhook Endpoint URL
       %td= @webhook_delivery.webhook_url
 
+    %tr
+      %th Redeliver
+      %td.actions
+        = link_to 'Redeliver', redeliver_client_application_webhook_delivery_path(params[:client_application_id], @webhook_delivery.id), method: :post, data: { confirm: "Are you sure?" }, class: 'btn btn-primary'
+
```

Comment:
Layout on the Show page:

<img width="800" alt="Screenshot 2024-10-30 at 5 18 49 PM" src="https://github.com/user-attachments/assets/b8a7d8aa-68f0-4200-ab2a-007e180bd894">